src/i18n/locales/en/translations.json

```diff
@@ -0,0 +1,25 @@
+{
+  "noInternetConnection": "No internet connection",
```

Comment:
@Brunonascdev can we make the json keys be the english sentence?
E.g.:
"No internet connection": "No internet connection",

So that we don't need to worry about complex json structures now. I understand how this could be valuable as we could apply different translations based on the context of the phrase, but I believe this is something we could potentially tackle in the future. Wdyt?